### ISSUE TITLE
GKvX: Mobile UI

### ISSUE THREAD

**Issue by Serg-Norseman** [Serg-Norseman/GEDKeeper#182]

**Comment by Serg-Norseman** [comment]
Xamarin
1. How to place an image from a resource in Button.Image?!!
2. How to make the Picker's text editable?!
3. Custom controls can only be with platform-dependent Renderers?!!

Why?! This is some kind of drivel...

**Comment by Serg-Norseman** [comment]
We need prototypes / sketches for design variants for dialogs, windows, components, color styles, fonts, borders and icons.

The most important first step: implementing any dialogue in a new redesign style.

I myself can’t create a new design, unfortunately, I don’t have any design abilities. But I can appreciate, because I have an inner vision of the product spirit.

Stages of implementation: Desktop WPF in the most complete possible package of features. Then - porting the developed XAML to UWP / Xamarin. In my opinion, subjective, but confident - UWP is a stunted direction (MS has already started promoting a new platform for UI - MAUI, how can it be? not a single completed and dominant - only decaying semi-finished products) and in its pure form is not worth the effort.

https://devblogs.microsoft.com/dotnet/introducing-net-multi-platform-app-ui/ :(

**Comment by altwarg** [comment]
@Serg-Norseman, I might suggest you to consider [Avalonia](https://github.com/AvaloniaUI/Avalonia) toolkit for a cross-platform UI application. It is mainly based on WPF. It is still in development, but as far as I concerned, it is close it the final release. About .NET MAUI -- no one can actually say when this will be ready. Knowing Microsoft's recent policy to OSS like PowerToys, Winget or WinUI/Fluent Design its could take yeast to make this toolkit production ready.

**Comment by Serg-Norseman** [comment]
@altwarg, I agree with the main part. But small experiments with Avalonia showed that I am not able to independently make a working prototype of my program on this framework. Two main problems:
1) I cannot transfer the entire set of functionality to the set of Avalonia components - when I looked at it, it had a noticeable specialization in the mobile direction, which is why some of the components that were important to me were simply not there. If I transfer "Modern GK" to Avalonia, then there will be the same obstacle as with Xamarin - some of the functionality will have to be cut, and the other will have to be restructured a lot. This is, in principle, superficial - a problem that I can solve.
2) The same problem, due to which WPF / UWP / Xamarin options are stagnant - I needs help, a person who understands design, to develop a coherent style of the future application in XAML. Own style, not borrowed. I went through about 2-3 dozen XAML themes posted on the Internet, and I could not find something suitable for GK spirit. I myself cannot develop one. I read tutorials on XAML, understood, I understand everything separately, but I can't create a style - I have no abilities.
